packages/chats/src/BubblePlainText.tsx

```diff
@@ -26,6 +28,29 @@ import { MessageStatus, OnLongPressText, OnPressUrl } from './types';
 import type { ChatPlainTextMessage, PlainTextMessage } from './types';
 import { useBubblePosition, useBubbleContainerStyle } from './useBubblePosition';
 import { useBubbleBackgroundColor, useBubbleRoundedCornerStyle } from './useBubbleStyle';
+import { URI_TON_SURF } from './constants';
+
+const renderParsedText = (matchingString: string, matches: string[]) => {
+    if (matches[1]) {
+        const domain = matches[0].replace(matches[1], '').replace(matches[2], '');
```

Comment:
Something is apparently wrong here..
First, matches[2] might be undefined
Second, matches refers not just to URLs, but it can be used with phones numbers, email, etc as per https://github.com/taskrabbit/react-native-parsed-text documentation

@savelichalex please, take a look